src/plots/map/constants.js

```diff
@@ -0,0 +1,264 @@
+var cartoPositronNoLabels = 'https://basemaps.cartocdn.com/gl/positron-nolabels-gl-style/style.json';
+var cartoDarkmatterNoLabels = 'https://basemaps.cartocdn.com/gl/dark-matter-nolabels-gl-style/style.json';
+var cartoVoyagerNoLabels = 'https://basemaps.cartocdn.com/gl/voyager-nolabels-gl-style/style.json';
+var arcgisSatHybrid = 'https://raw.githubusercontent.com/go2garret/maps/main/src/assets/json/arcgis_hybrid.json';
+var arcgisSat = {
+    version: 8,
+    name: 'orto',
+    metadata: {},
+    center: [
+        1.537786,
+        41.837539
+    ],
+    zoom: 12,
+    bearing: 0,
+    pitch: 0,
+    light: {
+        anchor: 'viewport',
+        color: 'white',
+        intensity: 0.4,
+        position: [
+            1.15,
+            45,
+            30
+        ]
+    },
+    sources: {
+        ortoEsri: {
+            type: 'raster',
+            tiles: [
+                'https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}'
+            ],
+            tileSize: 256,
+            maxzoom: 18,
+            attribution: 'ESRI &copy; <a href=\'http://www.esri.com\'>ESRI</a>'
+        },
+        ortoInstaMaps: {
+            type: 'raster',
+            tiles: [
+                'https://tilemaps.icgc.cat/mapfactory/wmts/orto_8_12/CAT3857/{z}/{x}/{y}.png'
+            ],
+            tileSize: 256,
+            maxzoom: 13
+        },
+        ortoICGC: {
+            type: 'raster',
+            tiles: [
+                'https://geoserveis.icgc.cat/icc_mapesmultibase/noutm/wmts/orto/GRID3857/{z}/{x}/{y}.jpeg'
+            ],
+            tileSize: 256,
+            minzoom: 13.1,
+            maxzoom: 20
+        },
+        openmaptiles: {
+            type: 'vector',
+            url: 'https://geoserveis.icgc.cat/contextmaps/basemap.json'
+        }
+    },
+
+    sprite: 'https://geoserveis.icgc.cat/contextmaps/sprites/sprite@1',
+    glyphs: 'https://geoserveis.icgc.cat/contextmaps/glyphs/{fontstack}/{range}.pbf',
+    layers: [
+        {
+            id: 'background',
+            type: 'background',
+            paint: {
+                'background-color': '#F4F9F4'
+            }
+        },
+        {
+            id: 'ortoEsri',
+            type: 'raster',
+            source: 'ortoEsri',
+            maxzoom: 16,
+            layout: {
+                visibility: 'visible'
+            }
+        },
+        {
+            id: 'ortoICGC',
+            type: 'raster',
+            source: 'ortoICGC',
+            minzoom: 13.1,
+            maxzoom: 19,
+            layout: {
+                visibility: 'visible'
+            }
+        },
+        {
+            id: 'ortoInstaMaps',
+            type: 'raster',
+            source: 'ortoInstaMaps',
+            maxzoom: 13,
+            layout: {
+                visibility: 'visible'
+            }
+        },
+    ],
+
+};
+
+var stylesMap = {
+    basic: cartoVoyager,
+    streets: cartoVoyager,
```

Comment:
@gvwilson @archmoj I feel we should remove the `streets` and `outdoor` style options for the `Map` trace types since we no longer have a distinct map type for them. However I'm not sure whether it's better to do it now or later. Maybe we could add a deprecation warning? Or simply remove them, and document that users will need to switch to another style when swapping `Mapbox` for `Map`.

Reply:
:+1: on a deprecation warning